README.md

```diff
@@ -112,6 +112,24 @@ apt-get --no-install-recommends -yqq install \
   xclip
 ```
 
+* For Ubuntu  >= 22.04
+
+```
+apt-get --no-install-recommends -yqq install \
+  bash-completion \
+  build-essential \
+  cmake \
+  libcurl4  \
+  libcurl4-openssl-dev  \
+  libssl-dev  \
+  libxml2 \
+  libxml2-dev  \
+  libssl3 \
```

Comment:
This is the noteworthy change, `libssl1.1` is no longer supported, but the tool builds fine with `libssl3`.

Reply:
Verified on 'Ubuntu 24.04'